conf/messages

```diff
@@ -1148,4 +1148,19 @@ itemBulkPackagingChoice.change.hidden = bulk packaging
 
 itemWineMoreInformationChoice.title = Do you want to give any more information about the wine?
 itemWineMoreInformationChoice.heading = Do you want to give any more information about the wine?
-itemWineMoreInformationChoice.error.required = Select yes if you would like to give more information about the wine
\ No newline at end of file
+itemWineMoreInformationChoice.error.required = Select yes if you would like to give more information about the wine
+
+itemWineGrowingZone.title = Which wine-growing zone has the wine come from?
+itemWineGrowingZone.heading = Which wine-growing zone has the wine come from?
+itemWineGrowingZone.hint.preLink = Select a wine-growing zone code. You can
+itemWineGrowingZone.hint.link = find more information about each wine-growing zone
+itemWineGrowingZone.hint.afterLink = in Annex IX to Regulation (EC) No 479/2008.
+itemWineGrowingZone.A = A
+itemWineGrowingZone.B = B
+itemWineGrowingZone.CI = CI
+itemWineGrowingZone.CII = CII
+itemWineGrowingZone.CIII_A = CIII(a)
+itemWineGrowingZone.CIII_B = CIII(b)
```

Comment:
According to the prototype this should be:
![image](https://github.com/hmrc/emcs-tfe-create-movement-frontend/assets/53828896/3344a6cb-5d79-49f0-934c-d8dc3a573455)
On the CYA page. However, I think what we have is right, maybe worth checking?

Reply:
I'm picking up the CYA task this week. So will confirm it as part of that. But that content I've added should be right.